server/api.go

```diff
@@ -1172,7 +1172,7 @@ func (a *API) dismissNotifications(w http.ResponseWriter, r *http.Request) {
 	// At this point, it might be /another/ post by the bot in the DM with that user, but we
 	// allow this for now.
 
-	post.Message = "To change your notification settings, open your user settings or run `/msteams notifications`"
+	post.Message = "To change your notification settings, select **Settings > MS Teams**, or run the **/msteams notifications** slash command."
```

Comment:
@cwarnermm, this message was added on clicking `Dismiss` after your review. I've inspired it based on the other text -- does it look good to you?

Reply:
Looks great, @lieut-data!